package.json

```diff
@@ -10,10 +10,13 @@
     "compile": "tsc",
     "test": "ava",
     "clean": "gts clean",
+    "cli": "node ./build/src/cli/index.js",
     "lint": "eslint './src/**/*.ts' './test/**/*.ts'"
   },
   "dependencies": {
-    "ava": "^3.10.0"
+    "@types/yargs": "^15.0.5",
```

Comment:
I don't think you need `@types/yargs` as a dependency as the types are only used for development. It can go in `devDependencies`.

Reply:
Done